### PR TITLE
Add a redirect header to send visitor to the new brand site

### PR DESCRIPTION
**Pull request by jeremy6d**
* Added a redirect header for `brand.adhoc.team` to the header template
* Upgraded `liquid` gem to 4.0.4 to incorporate a fix

### CODE REVIEW

**Comment by jjulian** [comment]
Can we separate out all the `localhost:4000` changes into it's own PR? Also, can that be parameterized using jekyll? Or maybe those urls should be relative?

**Comment by jjulian** [comment]
> Can we separate out all the `localhost:4000` changes into it's own PR? Also, can that be parameterized using jekyll? Or maybe those urls should be relative?

nevermind, more commits added :)

**Comment by jeremy6d** [comment]
Sorry, @jjulian, I fixed it before I saw your comment! Thanks for the catch.

File: _site/sitemap.xml [hunk 1]
```diff
@@ -20,66 +20,66 @@
 </url>
 <url>
 <loc>https://styleguide.adhoc.team/assets/img/content/logos/green-white-full-mark.pdf</loc>
-<lastmod>2020-05-19T17:11:25-04:00</lastmod>
+<lastmod>2023-08-29T10:32:09-04:00</lastmod>
```
Comment (jjulian) [inline _site/sitemap.xml:23]: really nerdy observation, but have these resources _actually_ changed?

**Pull request merged**